src/editor-state-prototypes.js

```diff
@@ -127,9 +127,9 @@ const stateSchema = {
       type: 'array',
       items: { type: 'string' },
     },
-    lastSaved: {}, // FIXME change to string ONLY with default 'never'
-    lastExport: {}, // FIXME change to string ONLY
-    sidePaneMode: {}, // FIXME change to string ONLY
```

Comment:
Should this be `{ type: 'string' }`?  Or is the FIXME obsolete and just being removed?